algorithms/unit_tests/TestBinSortA.hpp

```diff
@@ -247,10 +247,10 @@ TEST(TEST_CATEGORY, BinSortEmptyView) {
   Kokkos::View<int*, ExecutionSpace> v("v", 0);
 
   // test all exposed public sort methods
-  ASSERT_NO_THROW(Sorter.sort(ExecutionSpace(), v, 0, 0));
-  ASSERT_NO_THROW(Sorter.sort(v, 0, 0));
-  ASSERT_NO_THROW(Sorter.sort(ExecutionSpace(), v));
-  ASSERT_NO_THROW(Sorter.sort(v));
+  Sorter.sort(ExecutionSpace(), v, 0, 0);
+  Sorter.sort(v, 0, 0);
+  Sorter.sort(ExecutionSpace(), v);
+  Sorter.sort(v);
```

Comment:
Add a comment that we are just checking that these functions don't throw. It's pretty obvious what these tests do when you see `ASSERT_NO_THROW` but once you remove the macro it's not clear anymore. In 6 months, we will wonder why these tests don't check anything.

Reply:
Done